contracts/PowerBank.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity 0.7.4;
+
+/* solhint-disable indent */
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155Holder.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "./RuleManager.sol";
+
```

Comment:
Would recommend adding a block describing the specific responsibilities that PowerBank has.